src/classes/Accounts.cls

```diff
@@ -79,16 +79,64 @@ public inherited sharing class Accounts extends fflib_SObjects {
         }
     }
 
-    private void setNameAndGreetingsToReplacementText() {
-        householdNamingService.setNameAndGreetingsToReplacementText(householdAccounts());
-    }
-
     public void onAfterUpdate() {
         if (settings.isAdvancedHouseholdNaming()) {
             renameHouseholdsNeedingNameUpdate();
         }
     }
 
+    public void setAccountTypeAndIndividualFieldValues() {
+        for (Account acc : (List<Account>) getRecords()) {
+            if (needsTypeAndIndividualFieldsSet(acc)) {
+                setTypeAndIndividualFields(acc);
+            }
+        }
+    }
+
+    public void setSystemAccountTypeAndIndividualFieldValues() {
+        for (Account acc : (List<Account>) getRecords()) {
+            if (shouldResetAccountTypeAndIndividualFields(acc)) {
+                resetAccountTypeAndIndividualFields(acc);
+            }
+        }
+    }
+
+    /******************************************************************************************************
+    * @description utility to return the recordtypeId for a new account, given the account model.
+    * @return Id. if none is explicitly specified, returns null.
+    */
+    public static Id rtIdForNewAccount() {
```

Comment:
I might have called this out elsewhere, but I've got 2 questions on these 2 methods:

- Do they need to be public?
- Do the need to be static?

If yes to both, I would make the `rtIdForAccountModel` method into a getter that saves the data in a static var so it doesn't need to run though the describe and subsequent logic on each call to the method.

Reply:
Yeah, this is a bit of an odd method - I moved these from `ACCT_IndividualAccounts_TDTM.cls` because it felt like they fit better here.  But I'm hesitant to change this further now - they are used in a few classes, and there is a bit of a larger refactor here that might come into play in the Address Management refactor WI coming up.  Classes are using these to set the RT on a new Account, but the Accounts class should probably have a method that that does that instead.  Let me know if you want me to dig into this further and try to start that refactor here.